libs/fluid-elements/button-group/src/lib/button-group-item/button-group-item.spec.ts

```diff
@@ -0,0 +1,279 @@
+/**
+ * @license
+ * Copyright 2020 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { FluidButtonGroupItem } from './button-group-item';
+import { SPACE, ENTER } from '@dynatrace/shared/keycodes';
+import {
+  dispatchKeyboardEvent,
+  dispatchFakeEvent,
+} from '@dynatrace/testing/browser';
+
+function tick(): Promise<void> {
+  return Promise.resolve();
+}
```

Comment:
ca you move this to the testing browser and add some jsdoc to it? What it does and why it is needed?